Sources/ComposableAuthorizationProvider/ComposableAuthorizationProvider.swift

```diff
@@ -21,8 +21,18 @@ public struct AuthorizationControllerClient {
     #endif
 
     public enum DelegateEvent: Equatable {
-        case register(ASAuthorizationAppleIDCredential)
-        case signIn(ASAuthorizationAppleIDCredential)
+        public static func == (lhs: AuthorizationControllerClient.DelegateEvent, rhs: AuthorizationControllerClient.DelegateEvent) -> Bool {
```

Comment:
Can you move this to the bottom of the declaration?  I'd like to see the cases at the top.